### PR TITLE
Passive sts SAML2.0 feature support sample and autotests

### PR DESCRIPTION
**Pull request by cgchamath**
Updated the PassiveSTSSample to send SAML2.0 token requests.

Testing the feature
--------------------------

Deploy the PassiveSTSSample application.
To request a SAML 2.0 token use the URL - http://localhost:8080/PassiveSTSSampleApp?samlv=2-0
This will generate a passive sts request by populating the wreq parameter to have the requested token type to be 2.0. 
In the same way you can request SAML 1.1 token with the URL - http://localhost:8080/PassiveSTSSampleApp/?samlv=1-1
If you did not specify any parameter by default it will generate a SAML 1.1 token.

### CODE REVIEW

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/sts/TestPassiveSTS.java [hunk 1]
```diff
@@ -1,9 +1,18 @@
 package org.wso2.identity.integration.test.sts;
 
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
```
Comment (Pushpalanka) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/sts/TestPassiveSTS.java]: According to code styling java package imports should be below.

File: modules/samples/passive-sts/passive-sts-client/PassiveSTSFilter/src/main/java/org/wso2/identity/passivests/sample/AuthFilter.java [hunk 2]
```diff
@@ -3,6 +3,7 @@
 import static org.apache.commons.lang.StringEscapeUtils.escapeHtml;
 
 import java.io.IOException;
+import java.net.URLEncoder;
```
Comment (Pushpalanka) [inline modules/samples/passive-sts/passive-sts-client/PassiveSTSFilter/src/main/java/org/wso2/identity/passivests/sample/AuthFilter.java]: java packages should come at lowest

Comment (cgchamath) [inline modules/samples/passive-sts/passive-sts-client/PassiveSTSFilter/src/main/java/org/wso2/identity/passivests/sample/AuthFilter.java]: Fixed

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/sts/TestPassiveSTS.java (hunk 1 above)
Comment (cgchamath) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/sts/TestPassiveSTS.java]: Fixed

**Pull request merged**